core/client/src/main/java/alluxio/client/block/AlluxioBlockStore.java

```diff
@@ -194,7 +194,7 @@ public OutputStream getOutStream(long blockId, long blockSize, WorkerNetAddress
     }
     // No specified location to write to.
     if (address == null) {
-      throw new RuntimeException(ExceptionMessage.NO_WORKER_AVAILABLE.getMessage());
+      throw new RuntimeException(ExceptionMessage.NO_SPACE_FOR_BLOCK_ON_WORKER.getMessage());
```

Comment:
There is a new error message when I run bin/alluxio runTests.  I find if the exception message has {0}, like NO_SPACE_FOR_BLOCK_ON_WORKER, then the result will show "java.lang.IllegalArgumentException: The message takes 1 arguments, but is given 0". 

Because in the getMessage method in ExceptionMessage.java,  Preconditions.checkArgument() will trigger this error. But I am confused why it happens. 

runTest BASIC CACHE_PROMOTE CACHE_THROUGH
2017-04-05 12:14:36,928 INFO  ThriftClientPool - Alluxio client (version 1.5.0-SNAPSHOT) is trying to connect with FileSystemWorkerClient 10.110.165.236/10.110.165.236:29998 @ {}
2017-04-05 12:14:36,938 INFO  ThriftClientPool - Client registered with FileSystemWorkerClient @ 10.110.165.236/10.110.165.236:29998
2017-04-05 12:14:36,965 INFO  ThriftClientPool - Alluxio client (version 1.5.0-SNAPSHOT) is trying to connect with FileSystemWorkerClient 10.110.165.236/10.110.165.236:29998 @ {}
2017-04-05 12:14:36,968 INFO  ThriftClientPool - Client registered with FileSystemWorkerClient @ 10.110.165.236/10.110.165.236:29998
2017-04-05 12:14:37,018 ERROR CliUtils - Exception running test: alluxio.examples.BasicOperations@46238e3f
java.lang.IllegalArgumentException: The message takes 1 arguments, but is given 0
	at com.google.common.base.Preconditions.checkArgument(Preconditions.java:92)
	at alluxio.exception.ExceptionMessage.getMessage(ExceptionMessage.java:235)
	at alluxio.client.block.AlluxioBlockStore.getOutStream(AlluxioBlockStore.java:197)
	at alluxio.client.block.AlluxioBlockStore.getOutStream(AlluxioBlockStore.java:231)
	at alluxio.client.file.FileOutStream.getNextBlock(FileOutStream.java:320)
	at alluxio.client.file.FileOutStream.write(FileOutStream.java:287)
	at alluxio.client.file.FileOutStream.write(FileOutStream.java:272)
	at alluxio.examples.BasicOperations.writeFile(BasicOperations.java:77)
	at alluxio.examples.BasicOperations.call(BasicOperations.java:63)
	at alluxio.examples.BasicOperations.call(BasicOperations.java:39)
	at alluxio.cli.CliUtils.runExample(CliUtils.java:51)
	at alluxio.cli.TestRunner.runTest(TestRunner.java:148)
	at alluxio.cli.TestRunner.runTests(TestRunner.java:123)
	at alluxio.cli.TestRunner.main(TestRunner.java:100)